.github/workflows/pr-check.yml

```
@@ -14,99 +14,99 @@ jobs:
-            - 'packages/wayfinder-core/**'
-          react:
-            - 'packages/wayfinder-react/**'
-            - 'packages/wayfinder-core/**'
-          any:
-            - '**'
+      - uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+
+      - name: Check changed files
+        uses: dorny/paths-filter@v2
```

actionlint (1.7.10)

[error] the runner of "dorny/paths-filter@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)